src/main/java/module/siadap/domain/SiadapRootModule.java

```diff
@@ -107,6 +108,9 @@ private SiadapRootModule() {
         super();
         setBennu(Bennu.getInstance());
         setNumber(0);
+        final User adminUser = User.findByUsername("ist23487");
+        Group.dynamic("SiadapCCAGroup").mutator().grant(adminUser).toPersistentGroup();
```

Comment:
Group.dynamic("SiadapCCAGroup").mutator().or(Group.nobody());
This will result in the group being created with no users.